.infra/Pulumi.prod.yaml

```diff
@@ -38,7 +38,7 @@ config:
     defaultImageRatio:
       secure: AAABAHpff5rGIhX2EmH5QBZx1br4VVzK+JHKbbAUjYRT
     defaultImageUrl:
-      secure: AAABAKp3/LfYHhnzhQxriGmUaylvjnk9iSyKy+FEFfdZs2hWBVjVN4DkW0samRYP+yDoT53+8/EmwSLzeWOWSP+Ap0A5N/llgF3uV+pw7Z6bVjFDYuyG7yl1I7CzXIehjMtDEA0/s2NqFEm6R7uHNcDuoeQgkVwDITmm/1uvxqvdfc5wsn2VrCPXrXMF1FaoefwRB5A6X6S+V+H74QfSB41i0ZkXd1EkGzy8fBIeCePCq8K7xMhzeMJLOCoZ7z4SYknmOY2QlyPyyyCX+NGIlwnClZgwJ+rBdS/OdXF2vbPZZAZMmJd56uutyF+Tc3czHwA87A1M9SAK+1B2TXbFi+CA6Q/V2soLabNzCMR/76LgsWxSDCrT1ZxmLKud8HZrpxcSCxY2dw+aJ5Sve0OfEr7f0FMvBwRzkGObJ1xa68TqbrWAX42tF1c24YoSJ9+1uEB/8V/axg5EbuqlsZ4ZAvhbLp8th17nnePn0vbWLprZ2Zkkf0z+yb8xruS3QDR6g1Hsj2qTE2hPUuFxvBinSdDyEYUdTJpNMYPdlWAZ3AqVReNuMTI1cpvuwsr/sDHck4kWpp1Ia+cKsGFtHK7BECXt0OFAfo0qtcvABLA/01feFJGXzxWPvpMq2TUikTPPkZNoAyN8TQVjAj7oTGQx5UVCQlGAql19s9scK4/YHg9A2Q9TK8swr5gJ3A7ehbF6uBINNRNT4Do620xp1kH1k0e7D6bE0YDfDcBVaeWHYm87dowZyUZEm1hThFjoeMoZq8Ug5WnyKUdGDC7qy5GBGxTQbF3jUbsTJGXla5LsUcoZcoDe4iS01mz5oObvuEK5/e3kWP4ZxQ==
+      secure: AAABAKCkWcsCDUbEOB4Q5lyIYX8oeg8W3k5H0LlTrCynjRkvs0E5YdvPGf+qODswdpdznXQDr5NTpIEu83o4yiJyIDT0jl8nZs42JJ6sHavKxTKSB1kat5VoErL4aMH/TMnmujAe3E15TgqDBqHAb5wivOob4VNBjlcMPB5N+9A19i+3A136KuN708Hv/vDy35LtWGKb96OjqiaML83XtfmYBb9X2Bbd8IHbT1cyrH+WxgEevUkujqTBkFnKnPlBTnb3ePe6y9L2V1tJV2itk8YyZr69s5B2Rn8SQwrDtBTAxaoXB8ZexlcIOic7gC9fkUD3UaFbMET7UskeSIZAjV6SNmuSW3jYhLNRfFpBa5z4racgiFGw1i8DxfZtlKAPaDjF+AMGj+LVVVBySUyW66/cGQFoDFsrXM7j5A7SnctrsWcB/uGiqXWy7MY4okqqWsqROntMrzLnNnfTLLaKrHCI9ctgOiY0IOsKDKhDHjrdMAq3GzDFk++UrCwzQlZjL3h4CEiX9fKNKhtwLpGUl9n24+tZfuX161tvYmBU++b4TdaSe7dYm+OzKpmE4BiXUs4F2YSNjxxbIzB2Yq/i++fi9p5/P5R5ueT1tbstTeKTaNaIKwQNmucP5ikB+X5b94MSvQlheX7AgahQn1GuOooQJkN1gwYrPBRvdhtJwvkYyNwWhKf+lQCU0Tnfh92EfrZWsM9lOHdna0zeww0YEglJgbS1A904ciM85Uud+mgZRklrTE5BJCCXcj7i+4UUgjXygbAJqU8BIkUkaTDPydUOuoUgE7nu997vKM8t/0iNx9KlR7g9EQdWyv2QOFLO1bvBy3TIy5JRQJ4YkTOJD7RxbFU/O4K2rBJQLhF8xcSyO5ONuKS2S+xARu4XhMxaxqxZZ8ybzCmKZlKREFWDUvadtZvwDBMgoaayuwVECPfH3koEJYbYVyu1Xm1EozTX3K7RADVTjR1bMG1SGPNjWy9zeR3phPK+KK1AjmaVcWCUKqjo5XSgTks9EsNUHKBS52GMpsSQ0ff6d5GBWBv6c5zSYuPSamgNsQyy4Lqta3rvZNQgqY2n45X/G0M0aZq0SY/28bdcY4nTO4kg75L843kzIomTlz2f6lsA1lF2ZIJyTDjRvEp8w0WUTtSmdOR4oetxBDaBY53XqseMq5PvT7Vu8fQBGVG5IshOodGwmoE3WLG9nmLdnjF6iIwOtGj1n13F38NWDUI1Y707N4ohvM67vvhbmi14sEZsQNPVNRkzRRAw84PMv6mCsZ1MtYC0ApYnmG7e1SmTO+ihu1tzILjusX6W3p+6UIAipiPgSjrR5RgvCCYCi5nzHmO0h362YafpwJ35X00ZAwEur1Pq17ioSlX0C4015mSx5iheXh57aqDoxfs2gzlbTVywJSiU44rcVI3tg5ApqDz2RwTksn2FvtYNYlf3X9va+hCAkVeJjnNRSjrYNpBHYM4kdu9LBoxBNb5mdZSerDXtLw5ZJJHLj+INb6hVhp7T0X0plyeZkeUoxWYOzOil5RJaVB1pXPOlzJQnHJxBXjQ0/0ic8Qh8CF3xkTjHGWRIMlRVNig3aRAogDSPbXXhSv5skWUNIHsLa0CN3JFkGSr5oRIr3Q8n9IgSLgoGc8x4uDhvoQJ68Kczq1wfc4K19PdXAhfo34492g+uvw5qqZBbtwTG1Ul5pEwGL6M9qJgZfdUVfBtMkVB8ml/rVJR6wUMSd8vpf/yN1835gmno3EcAk1TGy3j2kQDphAwbfIRobpYC0Pc9VswNnHI0Qg2wZF7a/TturnCeNI005A==
```

Comment:
Doubt that this value needs to be stored as secure, especially since the same value is in adhoc 😬

Same goes with other non-secret "secrets" really. Thoughts @capJavert @idoshamun?

Reply:
Probably not, but wont matter much right?

Reply:
Nope, doesn't really matter. If we decide we want to change these we can change anytime.

Reply:
Nice to have not a blocker :)